pkg/scheduler/core/generic_scheduler.go

```diff
@@ -64,10 +64,9 @@ func (g *genericScheduler) Schedule(ctx context.Context, placement *policyv1alph
 	}
 
 	if len(feasibleClusters) == 0 {
-		return result, &framework.FitError{
-			NumAllClusters: clusterInfoSnapshot.NumOfClusters(),
-			Diagnosis:      diagnosis,
-		}
+		warning := &framework.FitError{NumAllClusters: clusterInfoSnapshot.NumOfClusters(), Diagnosis: diagnosis}
+		klog.Warningf("Couldn't finding feasible clusters: %v", warning)
```

Comment:
What is the log output form of this line? Do we need to use `warning.String()`?

Reply:
The outlog is:
```
W1207 08:46:59.576229       1 generic_scheduler.go:68] Couldn't finding feasible clusters: 0/3 clusters are available: 3 cluster(s) didn't match the placement cluster affinity constraint.
```

Reply:
ok, thx